services/motion/builtin/move_request.go

```diff
@@ -718,13 +716,10 @@ func (ms *builtIn) createBaseMoveRequest(
 	// current & desired orientation
 	if valExtra.motionProfile == motionplan.PositionOnlyMotionProfile {
 		if spatialmath.PoseAlmostCoincidentEps(goal.Pose(), startPose, motionCfg.planDeviationMM) {
-			return nil, errGoalWithinPlanDeviation
-		}
-	} else {
-		if spatialmath.OrientationAlmostEqual(goal.Pose().Orientation(), spatialmath.NewZeroPose().Orientation()) &&
-			spatialmath.PoseAlmostCoincidentEps(goal.Pose(), startPose, motionCfg.planDeviationMM) {
-			return nil, errGoalWithinPlanDeviation
+			return nil, motion.ErrGoalWithinPlanDeviation
 		}
+	} else if spatialmath.PoseAlmostEqualEps(goal.Pose(), startPose, motionCfg.planDeviationMM) {
```

Comment:
Do we not need `spatialmath.OrientationAlmostEqual(goal.Pose().Orientation(), spatialmath.NewZeroPose().Orientation())` to be true as well?

Reply:
`PoseAlmostEqualEps` is the logical AND of these two functions, so just a small simplification here